zebra-consensus/src/errors.rs

```diff
@@ -0,0 +1,44 @@
+//! Consensus validation errors
+
+use thiserror::Error;
+
+#[derive(Error, Debug)]
+pub enum TransactionError {
+    #[error("no transactions")]
+    NoTransactions,
+
+    #[error("first transaction must be coinbase")]
+    CoinbasePosition,
+
+    #[error("coinbase input found in non-coinbase transaction")]
+    CoinbaseInputFound,
+}
+
+#[derive(Error, Debug)]
+pub enum BlockError {
+    #[error("invalid transaction")]
```

Comment:
lets clarify that this error is about blocks
```suggestion
    #[error("block contains invalid transactions")]
```

Reply:
https://github.com/ZcashFoundation/zebra/pull/1070/commits/0858b9f3dfe4c093b62d0d88e9d8a20edc73d59f